From c1c9ba4348338afe21cf3a3e531409a80bb8cb1d Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@hadrons.org>
Date: Tue, 5 Mar 2024 02:47:50 +0100
Subject: [PATCH libaio 20/26] Rename sigset struct from data to aio_sigset

This makes the purpose more clear, and matches the naming in the aio
syscall implementation in Linux.

Signed-off-by: Guillem Jover <guillem@hadrons.org>
---
 src/io_pgetevents.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/io_pgetevents.c b/src/io_pgetevents.c
index 9d66dc8..63ff806 100644
--- a/src/io_pgetevents.c
+++ b/src/io_pgetevents.c
@@ -36,14 +36,14 @@ int io_pgetevents(io_context_t ctx, long min_nr, long nr,
 	struct {
 		unsigned long ss;
 		unsigned long ss_len;
-	} data;
+	} aio_sigset;
 
 	if (aio_ring_is_empty(ctx, timeout))
 		return 0;
 
-	data.ss = (unsigned long)sigmask;
-	data.ss_len = _NSIG / 8;
-	return __io_pgetevents(ctx, min_nr, nr, events, timeout, &data);
+	aio_sigset.ss = (unsigned long)sigmask;
+	aio_sigset.ss_len = _NSIG / 8;
+	return __io_pgetevents(ctx, min_nr, nr, events, timeout, &aio_sigset);
 }
 #else
 int io_pgetevents(io_context_t ctx, long min_nr, long nr,
-- 
2.43.0

